src/styles.css

```diff
@@ -175,3 +175,11 @@ i.code {
 div[data-lexical-editor='true'] {
   @apply outline-none;
 }
+
+.virtual-list {
+  @apply overflow-y-hidden;
+}
+
+.virtual-list > div > div {
+  @apply divide-y-[1px] dark:divide-gray-700/80;
+}
```

Comment:
We have the same class in `Card` component as well? Do we need it at both places?

Reply:
We are not using card inside virtual list and single publication, so we need this 🙇🏼